Makefile

```diff
@@ -188,6 +188,9 @@ test/addons/.buildstamp: config.gypi \
 # TODO(bnoordhuis) Force rebuild after gyp update.
 build-addons: $(NODE_EXE) test/addons/.buildstamp
 
+clear-stalled:
+	ps awwwx | grep Release/node | grep -v grep | awk '{print $1}' | xargs kill
```

Comment:
What about simply using `pkill` instead? Isn't that available on all *nix platforms in CI?